e2e/pages/ward-allocation.ts

```diff
@@ -0,0 +1,53 @@
+import { type Page } from '@playwright/test';
```

Comment:
Nice to see the use of the page object model here. The file naming convention we follow for these pages uses the `-page` suffix. Let's rename this to `ward-page.ts`.